gui/qt/transaction_dialog.py

```diff
@@ -296,8 +296,14 @@ def format_amount(amt):
         o_text.clear()
         cursor = o_text.textCursor()
         for addr, v in self.tx.get_outputs():
-            cursor.insertText(addr.to_ui_string(), text_format(addr))
+            addrstr = addr.to_ui_string()
+            cursor.insertText(addrstr, text_format(addr))
             if v is not None:
-                cursor.insertText('\t', ext)
+                if len(addrstr) > 42: # for long outputs, make a linebreak.
+                    cursor.insertBlock()
+                    addrstr = "   ^^^"
```

Comment:
Is there any chance we can use the unicode character U+23CE (RETURN SYMBOL, ⏎) or another character more visual?
[This slackoverflow comment](https://stackoverflow.com/a/18931703/3954464) is apropos.

Reply:
Fantastic suggestion... here are some other ones ↳ ↪ ➥ ⤷

Reply:
You planning on adding this symbol? or? (I'm waiting for that to be added before merging..)